### DISCUSSION TITLE
thinkphp8 支付宝h5支付返回的表单如何获取

### DISCUSSION THREAD

**Discussion by songkaixuan** [yansongda/pay#956] [Q&A] [answered]
包为 3.5.2 版本，接口返回的表单如何获取到，作为一个参数返给前端

**Comment by songkaixuan** [comment, marked as answer]
已解决 $result     = (string)$result->getBody();

**Comment by syssts** [comment]
请问下，我也是thinkphp8 为啥有config,会提示错误
![image](https://github.com/user-attachments/assets/974927d8-c602-41b1-8027-f1c7f19e686c)
我明明能打印出$config的内容
![image](https://github.com/user-attachments/assets/7d398b4a-2bc3-4449-ae9a-873a12b8a838)

**Reply by qianye4746** [reply]
尝试把“wechat”也写在配置数组里：['wechat' => ["defult" => [你的配置数组]]]  这一整组都初始化进去，而不是只初始化['defult' => [你的配置数组]]
